app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,39 @@
+/*
+  Place all the styles related to the matching controller here.
+  They will automatically be included in application.css.
+*/
+
+.comments-form {
+  margin-top: -25px;
+  margin-bottom: 70px;
+}
+
+.comments {
+  margin: 30px 0 70px 0;
+  background: #EBEBEB;
+  padding: 15px 20px 20px 20px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `15px 20px 20px` instead of `15px 20px 20px 20px`